Helpers/MarkerManager.h

```diff
@@ -0,0 +1,39 @@
+//
+//  MarkerManager.h
+//  social-maps
+//
+//  Created by Bevin Benson on 7/27/18.
+//  Copyright © 2018 Bevin Benson. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMaps/GoogleMaps.h>
+#import <GooglePlaces/GooglePlaces.h>
+#import <Parse/Parse.h>
+#import "Marker.h"
+
+extern NSString *const kFavoritesKey;
+extern NSString *const kWishlistKey;
+extern NSString *const kFollowFavKey;
+
+@interface MarkerManager : NSObject
+
+@property (strong, nonatomic) NSMutableDictionary<NSString*, NSMutableArray<GMSMarker*>*> *markersByPlaceCategory;
```

Comment:
can you make these immutable arrays instead?

Reply:
WOOPS SORRY

I mean can you switch the mutable arrays to be immutable arrays, and mutable dictionaries to be immutable dictioanries?